src/plugins/interactive_setup/server/elasticsearch/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { ElasticsearchService, EnrollResult } from './elasticsearch_service';
```

Comment:
nit: I'd prefer to keep the folder structure flat here. Nesting individual files only adds unnecessary work.